src/js/controllers/mapController.ts

```diff
@@ -30,16 +30,16 @@ export class MapController {
       .when(
         () => {
           console.log('mapview is loaded');
-          store.dispatch({ type: 'MAP_READY', isMapReady: true });
+          store.dispatch({ type: 'MAP_READY', payload: true });
```

Comment:
@vaidotasp I noticed `mapview/reducers.ts` conditionally returns an object with the property `isMapReady` or `loadError`. I was thinking updating the property from `payload` to `isMapReady` or `loadError` could keep the naming consistent across the app, which could be helpful with debugging down the line